### DISCUSSION TITLE
o1-preview and o1-mini support

### DISCUSSION THREAD

**Discussion by juan-abia** [langfuse/langfuse#3338] [Ideas]
### Describe the feature or potential improvement

Add this new models to langfuse default models. Reasoning tokens are not available trough the API, but we do get the number of tokens used (see image)
![image](https://github.com/user-attachments/assets/bf1cf34d-032d-4e36-9326-c38db99e3640)

**Comment by marcklingen** [comment]
working on this, we are tracking this here: https://github.com/langfuse/langfuse/issues/3340

**Comment by clemra** [comment]
let's gooo!

**Comment by juan-abia** [comment]
Can we expect a new release today? thanks for the speed btw, you rock guys ❤️

**Reply by marcklingen** [reply]
yes!
